DTW/Bridge of Doom DTF/map.json

```diff
@@ -0,0 +1,96 @@
+{
+	"name": "Bridge Of Doom DTF",
+	"authors": ["Clijmart"],
+	"version": "1.0",
+	"gametype": "DTM",
+	"teams": [
+		{
+			"id": "yellow",
+			"name": "Bennyvil",
+			"color": "yellow",
+			"max": 50,
+			"min": 1
+		},
+		{
+			"id": "gold",
+			"name": "Evilucas",
+			"color": "gold",
+			"max": 50,
+			"min": 1
+		}
+	],
+	"spawns": [
+		{ "teams": ["spectators"], "coords": "0.5, 51, 0.5, 0" },
+		{ "teams": ["yellow"], "coords": "55.5, 27, -36.5, 45" },
+		{ "teams": ["gold"], "coords": "-54.5, 27, 37.5, -135" }
+	],
+	"dtm": {
+		"monuments": [
+			{
+				"name": "Evilucas",
+				"teams": ["gold"],
+				"materials": ["wool"],
+				"region": {
+					"min": "-67, 62, 8",
+					"max": "-73, 56, 2"
+				},
+				"health": 20
+			},
+			{
+				"name": "Bennyvil",
+				"teams": ["yellow"],
+				"materials": ["wool"],
+				"region": {
+					"min": "67, 62, -8",
+					"max": "73, 56, -2"
+				},
+				"health": 20
+			}
+		]
+	},
+	"killstreaks": [{
+			"count": 1,
+			"repeat": true,
+			"commands": ["give %killername% minecraft:golden_apple 1",
+				"give %killername% minecraft:arrow 4"
+			]
+		}
+	],
+	"kits": [
+		{
+			"name": "Default",
+			"items": [
+				{"type": "item", "material": "stone sword", "slot": 0, "unbreakable": true},
+				{"type": "item", "material": "bow", "slot": 1, "unbreakable": true},
+				{"type": "item", "material": "iron pickaxe", "slot": 2, "unbreakable": true},
+				{"type": "item", "material": "stone axe", "slot": 3, "unbreakable": true},
+				{"type": "item", "material": "log", "slot": 4, "amount": 64},
+
```

Comment:
Here too, try organizing it so it's 4 items (empty line), 3 items (empty line), 4 items.